canary/canary.csproj

```diff
@@ -33,6 +33,20 @@
     <None Include="$(SpaRoot)**" Exclude="$(SpaRoot)node_modules\**" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\vrdr-dotnet\VRDR.Messaging\VRDRMessaging.csproj" />
+    <ProjectReference Include="..\..\vrdr-dotnet\VRDR\DeathRecord.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>

```

Comment:
You should be able to use the project references directly in the .NET portion, or did you run into version incompatibility?